spec/services/events/validate_creation_service_spec.rb

```diff
@@ -179,6 +179,34 @@
         end
       end
 
+      context 'when field_name value is not a number' do
+        let(:billable_metric) { create(:sum_billable_metric, organization: organization) }
+        let(:params) do
+          {
+            code: billable_metric.code,
+            external_customer_id: customer.external_id,
+            properties: {
+              item_id: 'test'
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.